### DISCUSSION TITLE
bruh why

### DISCUSSION THREAD

**Discussion by Riskerman21** [KadeArchive/Kade-Engine#1703] [Q&A] [answered]
why does this code exist huh:

                        case 'Blammed':
                            {
                                if (curBeat > 30 && curBeat < 190)
                                {
                                    if (curBeat < 90 || curBeat > 128)
                                    {
                                        if (curBeat % 4 == 2)
                                        {
                                            if (!triggeredAlready)
                                            {
                                                gf.playAnim('cheer');
                                                triggeredAlready = true;
                                            }
                                        }
                                        else
                                            triggeredAlready = false;
                                    }
                                }
                            }

**Comment by EcoVeni** [comment, marked as answer]
ask the developers lol

**Reply by raf13lol** [reply]
yes

**Comment by libracrumbs** [comment]
Its for the animation of GF, when the bf does the Peace sign (idk what to call it)

**Comment by Gostar64** [comment]
I'm pretty sure it exists to make girlfriend cheer when the (HEY!) sounds play in the instrumental of Blammed

**Reply by Riskerman21** [reply]
The arguments give null it doesn't exist man. Girlfriend doesn't cheer in blammed

**Reply by Gostar64** [reply]
![image](https://user-images.githubusercontent.com/88996224/129615817-77533f4c-3b0c-49f4-9cca-310a8ea665d7.png)
She does in 1.6.2

**Reply by StuffyAnt** [reply]
she has cheered ever since 1.4.2ish
